Source/WebCore/svg/SVGFEComponentTransferElement.cpp

```diff
@@ -106,27 +106,24 @@ bool SVGFEComponentTransferElement::setFilterEffectAttributeFromChild(FilterEffe
     auto& feComponentTransfer = downcast<FEComponentTransfer>(effect);
     auto& child = downcast<SVGComponentTransferFunctionElement>(childElement);
 
-    if (attrName == SVGNames::typeAttr)
+    switch (attrName.nodeName()) {
+    case AttributeNames::typeAttr:
```

Comment:
Seems to me these functions are extra wordy because of the long local variable names. I would rename the arguments to `filterEffect` and then use `effect` for the downcast result local instead of unwieldy long names like `feComponentTransfer`.